<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ms_score</class>
 <widget class="QDialog" name="ms_score">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>分数表</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,8,1">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,4,1,2,8,1,4,1,4,1,4,1,4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>筛选:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbox_col">
        <item>
         <property name="text">
          <string>按类型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>学号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>姓名</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>学期</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>课程名称</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>分数</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>班级号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>教师</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>43</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>关键字:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_keyword">
        <property name="placeholderText">
         <string>请输入关键字</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>42</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_search">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>43</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="text">
         <string>新增</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_modify">
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>43</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_delete">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="9,1">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_return">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
